python/py_vapid/tests/test_vapid.py

```diff
@@ -24,8 +24,11 @@
 -----END PUBLIC KEY-----
 """
 
-T_PUBLIC_RAW = """EJwJZq_GN8jJbo1GGpyU70hmP2hbWAUpQFKDBy\
-KB81yldJ9GTklBM5xqEwuPM7VuQcyiLDhvovthPIXx-gsQRQ==""".strip('=')
+# this is a DER RAW key ('\x04' + 2 32 octet digits)
+T_PUBLIC_RAW = (
+    "BBCcCWavxjfIyW6NRhqclO9IZj9oW1gFKUBSgwcigfNc"
+    "pXSfRk5JQTOcahMLjzO1bkHMoiw4b6L7YTyF8foLEEU"
+    ).strip('=')
```

Comment:
Was planning on keeping it. It just turned out that key generation didn't include padding, but we want to make sure it isn't included in any future keys.